File: Array/python/Find duplicate in Array.py
class Solution:
    # @param A : tuple of integers
    # @return an integer
    def repeatedNumber(self, A):
        check = 0
        for i in A:
            if(check&(1<<i)>0):
                return i
            else:
                check|=(1<<i)
        return -1

File: Arrays and Matrix/c++/FInd duplicate in Array.cpp
int Solution::repeatedNumber(const vector<int> &A) {
    
    int check = 0; // can be implemented using the checker bit : as in python code
    vector<int>temp(A); // Using O(n) space
    sort(temp.begin(),temp.end());
    
    for(int i=0;i<temp.size()-1;i++)
    {
        if(temp[i]==temp[i+1])
        {
            return temp[i];
        }
    }
    return -1;
}
